library/src/loader_impl.cxx

```diff
@@ -199,10 +195,14 @@ loader& loader_impl::loadGeometry(const std::string& filePath, bool reset)
   this->Internals->GenericImporter->RemoveObservers(vtkCommand::ProgressEvent);
   progressWidget->Off();
 
-  if (this->Internals->Interactor)
+  // Initialize the animation using temporal information from the importer
+  this->Internals->AnimationManager.Initialize(&this->Internals->Options, &this->Internals->Window,
+    this->Internals->Interactor, this->Internals->GenericImporter);
+
+  double animationTime = this->Internals->Options.getAsDouble("scene.animation.time");
+  if (animationTime != 0)
```

Comment:
Since we can only know if an option was changed or if it is default by comparing the default value, this has a very small side effect that if an animation has a time range that include 0 inside, we wont be able to load the time value zero, it will just be as if no time value was set, which can be different with the generic importer and certain specificaly crafted exodus file.

I dont think it matters and I have no fix anyway, at least until we have a concept of "isDefault" in the options.